src/shared/reducers/app.ts

```diff
@@ -33,7 +33,7 @@ const initialState: AppState = {
     showTemplateControlBar: false,
     timeZone: 'Local',
     navBarState: 'collapsed',
-    scriptQueryBuilder: false,
+    scriptQueryBuilder: true,
```

Comment:
This should be the default now. Was never done properly -- uncovered while testing.

Reply:
Setting this to `true` as default will cause the [flux function injection failure ](https://app.circleci.com/pipelines/github/influxdata/monitor-ci/19356/workflows/70d227d8-1656-4553-8f02-147925199b95/jobs/1373051/tests#failed-test-0) in old data explorer since the old style will be replaced by the new style. 

https://github.com/influxdata/ui/blob/43c27ee951dc258f762a04bd9cb2d0b113d3c97c/src/shared/components/FilterList/InjectionOption.tsx#L99-L108

For the TSM org users and pre-marty release IOx org users, the old data explorer is still the default option unless they turn off the toggle "Switch to the old Data Explorer". 

I think we should set this value back to `false` for now, then turn back on when the old data explorer is completely gone

Reply:
Worked with @appletreeisyellow , and we are instead going to do a proper fix:
* leave `scriptQueryBuilder: true` since it's the proper, intended behavior
* Make sure we don't use the feature flag `showOldDataExplorerInNewIOx`, since it forces the view state out of alignment with this `AppSettings.scriptQueryBuilder`
* instead, make use of a new fixture `cy.mockIsCloud2Org()`, followed by `cy.switchToDataExplorer()`